notebooks/CubevizExample.ipynb

```diff
@@ -63,7 +59,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "viz.show_in_new_tab()"
+    "viz.show(loc='sidecar:tab-after')"
```

Comment:
See above.

Reply:
Removed these cells in all the notebooks!